llvm/lib/Target/AMDGPU/AMDGPUMacroFusion.cpp

```diff
@@ -54,13 +52,7 @@ static bool shouldScheduleAdjacent(const TargetInstrInfo &TII_,
   return false;
 }
 
-} // end namespace
-
-
-namespace llvm {
-
-std::unique_ptr<ScheduleDAGMutation> createAMDGPUMacroFusionDAGMutation() {
+std::unique_ptr<ScheduleDAGMutation>
```

Comment:
the namespace clean-up can go to another commit

Reply:
Fair enough, I reverted this file.

Reply:
There are also changes in several files related to namespace cleanup.